pepper-apis/src/main/java/org/broadinstitute/ddp/util/TestRedisConnection.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.ddp.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+public class TestRedisConnection {
```

Comment:
How about `RedisConnectionValidator`?  Not sure if other folks are using their own class name regexes for things like intellij test profiles--starting or ending a class name with `Test` might be confusing.

Reply:
1ff7ed3e02d726d6247aab104ad3151737017fa5